readme.md

```diff
@@ -32,6 +32,37 @@ unicorn
 */
 ```
 
+## API
+
+### stripIndent(string, options?)
+
+Returns a `string`.
+
+#### options
+
+Type: `object`
+
+##### indent
+
+Type: `number`\
+Default: The line with the least amount of leading whitespace, ignoring empty lines, determines the number.
+
+Whitespace number to be stripped.
+
+## Advanced usage
+
+Passing `indent` option: useful if the string may contain only whitespaces (like manipulating `node.raws.before` in a [PostCSS](https://github.com/postcss/postcss) plugin) or for optimization. Check the indentation first for the context using e.g. [min-indent](https://github.com/jamiebuilds/min-indent) and use this module to just strip it.
```

Comment:
It's not clear from what you've written here that the point is to use a different source to analyze the indent than what it's being applied to.

I would also mention the use-case:

> optimize multiple runs, checking the indentation number only once and then only applying.